lib/routes/wifi/index.ts

```diff
@@ -0,0 +1,81 @@
+import * as _ from 'lodash';
+import {NetworkManager} from '../../nm';
+
+export default {
+	'connect-network': {
+		method: 'POST',
+		handler: (nm: NetworkManager) => (req: any, res: any) => {
+			const connection = req.body.value;
+			nm.connectNetwork(connection)
+			.then(() => {
+				return res.status(200).json();
+			})
+			.catch((err) => {
+				return res.status(err.type).json(err);
+			});
+		},
```

Comment:
```typescript
try {
    await nm.connectNetwork(connection)
    return res.status(200).end();
} catch (err) {
    res.status(err.type).json(err);
}
```

Reply:
Is `err.type` an HTTP status code in all cases?

Reply:
As of now, it is, mainly for convenience